src/components/Home/Map/IncidentFocus.js

```diff
@@ -36,7 +36,7 @@ export default function IncidentFocus({ zoomOnCluster }) {
         >
           <Panel
             className="collapseText"
-            header="Select to view Incidents"
+            header="View Incident Reports"
```

Comment:
Think we should try to keep the "Select to " wording, but I understand that it creates a new bug to fix

Reply:
I agree. We should include the wording of "Select to..." but at the moment the text is too long and creates a new line out of the current drop-down box. We have a Trello card to work on those divs for the map, so when we work on that we should make that change.